test/upgrade/prober/configuration.go

```diff
@@ -34,17 +36,82 @@ import (
 )
 
 const (
-	configName       = "wathola-config"
-	configMountPoint = "/home/nonroot/.config/wathola"
-	configFilename   = "config.toml"
-	watholaEventNs   = "com.github.cardil.wathola"
-	healthEndpoint   = "/healthz"
+	defaultConfigName          = "wathola-config"
+	defaultConfigHomedirPath   = ".config/wathola"
+	defaultHomedir             = "/home/nonroot"
+	defaultConfigFilename      = "config.toml"
+	defaultWatholaEventsPrefix = "com.github.cardil.wathola"
+	defaultBrokerName          = "default"
+	defaultHealthEndpoint      = "/healthz"
+	defaultFinishedSleep       = 5 * time.Second
 )
 
-var (
-	eventTypes = []string{"step", "finished"}
-	brokerName = "default"
-)
+var eventTypes = []string{"step", "finished"}
+
+// Config represents a configuration for prober.
+type Config struct {
+	Wathola       WatholaConfig
```

Comment:
Would it make sense to inline/embed this so that we don't have so many levels of inspection

Reply:
How you would like to have it? Can you give me an example?

`p.config.Wathola.Config.MountPoint` -> `p.config.ConfigMapMountPoint` ?

In my opinion the first one is more descriptive, and got Go struct comments.

Reply:
Yes basically just remove the Wathola name so when we use the struct don't have so many named levels of indirection, kind of like k8s typemeta and objectmeta